app/assets/stylesheets/application.scss

```diff
@@ -54,3 +54,9 @@ $hmrc-assets-path: "/report-quarterly/income-and-expenses/view/assets/lib/hmrc-f
   width: 68%;
 }
 
+.govuk-insettext-zero-spacing {
+    margin-top: 0px;
+    border-left: 0px;
+    padding: 0px;
+    margin-bottom: 1.05263em;
```

Comment:
we shouldn't mix px and ems - there are existing hmrc classes for spacing (please use): https://design-system.service.gov.uk/styles/spacing/